src/Evolution/DgSubcell/SubcellOptions.hpp

```diff
@@ -120,20 +127,46 @@ class SubcellOptions {
         "phenomenon."};
   };
 
+  /// \brief A list of block names on which to never do subcell.
+  ///
+  /// Can overlap with blocks listed in `OnlyDgBlockGroups`.
+  ///
+  /// Set to `None` to allow subcell in all blocks.
+  struct OnlyDgBlocksAndGroups {
```

Comment:
I think the `AndGroups` isn't necessary. It's mentioned in the help string.

Reply:
I disagree, I think it makes it easier to read the input file